### DISCUSSION TITLE
This Repo mess with the original AUTO's Env?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [vladmandic/sdnext#359] [Q&A] [answered]
Because now when i start the Original AUTO's i get this **FAVICON** in the browser:

![Screenshot_4](https://user-images.githubusercontent.com/13344308/233753086-1c79de23-e6a6-4ac0-97f1-29797cd7b84f.png)

I installed this repo via the .BAT, i didn't executed any .py.

**Comment by vladmandic** [comment, marked as answer]
It doesn't touch original at all.
But that's how windows works, if there is no specific icon assigned, it will pick something on its own. This repo has icon, original doesn't.

**Reply by ZeroCool22** [reply]
> It doesn't touch original at all. But that's how windows works, if there is no specific icon assigned, it will pick something on its own. This repo has icon, original doesn't.

Thx.
